### PR TITLE
Enabled skipped Identity test 

### PR DESCRIPTION
**Pull request by emdobrin**

### CODE REVIEW

File: Sources/EdgeNetworkHandlers/EdgeHitProcessor.swift [hunk 1]
```diff
@@ -22,6 +22,7 @@ class EdgeHitProcessor: HitProcessing {
     private var getSharedState: (String, Event?) -> SharedStateResult?
     private var readyForEvent: (Event) -> Bool
     private var entityRetryIntervalMapping = ThreadSafeDictionary<String, TimeInterval>()
+    var batchLimit: Int = 0
```
Comment (nporter-adbe) [inline Sources/EdgeNetworkHandlers/EdgeHitProcessor.swift:25]: let's make this a `let`

**Review by nporter-adbe — APPROVED** [review]
one minor comment

File: Sources/EdgeNetworkHandlers/EdgeHitProcessor.swift (hunk 1 above)
Comment (nporter-adbe) [reply Sources/EdgeNetworkHandlers/EdgeHitProcessor.swift:25]: oh actually, the protocol requires a setter. ignore this comment.

Comment (emdobrin) [reply Sources/EdgeNetworkHandlers/EdgeHitProcessor.swift:25]: I am getting this error `Type 'EdgeHitProcessor' does not conform to protocol 'HitProcessing' Candidate is not settable, but protocol requires it`. I see `var batchLimit: Int { get set} ` in the protocol. I guess we only need the getter, do we use the setter somewhere else?

Comment (nporter-adbe) [reply Sources/EdgeNetworkHandlers/EdgeHitProcessor.swift:25]: Yeah my bad, I forgot we require a setting on the batch limit. I believe this is mainly to support if the batch limit is updated in Analytics via config updates.

**Pull request merged**